configure.py

```diff
@@ -601,9 +598,11 @@ def _ffmpeg_setup(cfg):
         "asetnsamples",
         "asettb",
         "copy",
+        "dynaudnorm",
         "format",
         "fps",
         "hflip",
+        "loudnorm",
```

Comment:
I'm curious about the addition of these 2 filters

Reply:
Mentioned in the commit message. They are quite useful if one wants to some some audio alongside video rendering.